Source/Blazorise/Interfaces/Modules/IJSOffcanvasModule.cs

```diff
@@ -0,0 +1,29 @@
+﻿#region Using directives
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+#endregion
+
+namespace Blazorise.Modules;
+
+
+/// <summary>
+/// Contracts for the <see cref="Offcanvas"/> JS module.
+/// </summary>
+public interface IJSOffcanvasModule : IBaseJSModule
```

Comment:
I think we don't need IJSOffcanvasModule at all. We can safely remove all JS Modules related to Offcanvas. Including `offcanvas.js` file.